src/Symfony/Component/String/Tests/AbstractAsciiTestCase.php

```diff
@@ -1067,6 +1067,7 @@ public static function provideSnake()
             ['x_y', 'x_y'],
             ['x_y', 'X_Y'],
             ['xu_yo', 'xu_yo'],
+            ['symfony_is_great', 'symfony-is-great'],
```

Comment:
Can you also add `['symfony_is_great', 'symfony.is.great'],` to account for https://github.com/symfony/symfony/issues/57612#issuecomment-2199940103?